src/components/metadata/list.js

```diff
@@ -108,6 +113,10 @@ class MetadataList extends React.PureComponent {
             onNext={this.handleNext}
             onPrev={this.handlePrev}/>
         )}
+        { this.hasNewMetadataField && <NewMetadataField
```

Comment:
Better to move the `<NewMetadataField` to the next line. We usually start the tag on a new line unless it's a one-liner.